gateways/db/accounts.go

```diff
@@ -0,0 +1,69 @@
+package db
+
+import (
+	"context"
+	"time"
+)
+
+// Database, metódos do banco de dados
+type DatabaseMethods interface {
+	AddedAccount(ctx context.Context) (Database, error)
+	ShowBalanceId(ctx context.Context) (Database, error)
+	ShowAccounts(ctx context.Context) ([]Database, error)
+	FindDocument(ctx context.Context) (Database, error)
+}
+
+type Database struct {
+	ID        string    `json:"id"`
+	Name      string    `json:"name"`
+	CPF       string    `json:"cpf"`
+	Secret    string    `json:"secret"`
+	Balance   int       `json:"balance"`
+	CreatedAt time.Time `json:"created_at,omitempty"`
+}
+
+// NewRepositoryDB, cria um novo repositório do banco
+func NewRepositoryDB(db DatabaseMethods) *Database {
+	return &Database{}
+}
+
+// addedAccount, insere a conta no banco
+func (db *Database) AddedAccount(ctx context.Context) ([]Database, error) {
```

Comment:
```suggestion
func (db *Database) AddAccount(ctx context.Context) ([]Database, error) {
```
`AddedAccount` passa uma ideia de pergunta em vez de ação. Poderia usar até um nome mais atrelado ao problema, por exemplo `RegisterAccount`, acha que faz sentido?

Reply:
Também tem a mesma observação do usecase, talvez você possa retornar somente o identificador da nova conta no banco de dados, no caso o índice no slice.